README.md

```diff
@@ -1,3 +1,7 @@
+# Issues for this repository are disabled
+
+Issues for this repository are tracked in Red Hat Jira. Please head to https://issues.redhat.com/browse/MAISTRA in order to browse or open an issue.
```

Comment:
https://github.com/microsoft/terminal/pull/8114/files might be a better approach. Use the process users are already used to and redirect them in that path rather than making them look for other info.

Reply:
It doesn't prevent users to just skip it and create issues if they want to.

Reply:
@brian-avery done